### DISCUSSION TITLE
How can I configure column to store only time with timezone?

### DISCUSSION THREAD

**Discussion by oleksandr-petryk** [sequelize/sequelize#14207] [Help & Questions]
I use postgres dialect. I did not found any data type for this.

**Comment by oleksandr-petryk** [comment]
I asked this question in stackoverflow, but no one answered me

**Comment by ephys** [comment]
[`DataTypes.TIME`](https://github.com/sequelize/sequelize/blob/main/src/data-types.js#L464) should be `TIME WITH TIME ZONE` by default in postgres
Edit: No, it's *without* time zone by default

**Reply by oleksandr-petryk** [reply]
I tried to use it, but when I try to save time with timezone without date it cause error. May be I make something wrong

**Reply by ephys** [reply]
What does the value you're trying to save look like?

**Reply by oleksandr-petryk** [reply]
10:00 GMT+0200

**Reply by ephys** [reply]
Try something like `'10:00+0200'` instead to see if it works

**Reply by oleksandr-petryk** [reply]
I try again and answer, thank you

**Reply by oleksandr-petryk** [reply]
I tried it, below model and service.

Service:
```js
  async testTime() {
    TestModel.create({
      time: '10:00+0200'
    })
      .then(r => {
        console.log(r.dataValues)
      })
      .catch(e => {
        console.log('Something went wrong')
      })
  }
```

Model:
```js
module.exports = (sequelize, DataTypes) => {
    const TestModel = sequelize.define(
        'TestModel',
        {
            id: {
                type: DataTypes.INTEGER,
                autoIncrement: true,
                primaryKey: true,
            },
            time: {
              type: DataTypes.TIME,
            }
        },
        {
            tableName: 'Test',
        }
    )

    return TestModel
}
```

It's table view:

![image](https://user-images.githubusercontent.com/57829569/157020750-6c805c80-8709-43cc-917e-c007bcb5acea.png)

Column time has type `time without time zone`

Probably it's type that I need:

![image](https://user-images.githubusercontent.com/57829569/157021976-eef52fdf-7e3b-43ab-9ebe-d3ac140d46b2.png)

See the type [here](https://www.postgresql.org/docs/9.5/datatype-datetime.html#DATATYPE_DATETIME "PostgreSQL website")

**Reply by oleksandr-petryk** [reply]
The only solution I found was to create my own data type.

**Reply by ephys** [reply]
Yep, I misread the pg documentation this morning and switched both versions
Looks like we don't have a built-in DataType for time with time zone 

Feature request welcome :)
